modules/framework/galasa-parent/dev.galasa.framework.api.users/src/main/java/dev/galasa/framework/api/users/internal/routes/UserUpdateRequestValidator.java

```diff
@@ -31,6 +31,12 @@ public void validateUpdateRequest(UserUpdateData userUpdateData) throws Internal
         String roleId = userUpdateData.getrole();
         if (roleId==null || roleId.trim().isBlank()) {
             // Role is blank, so the user isn't trying to update the role field in the user record. Ok.
+        } 
+        else if (roleId == "3") { // Assigning a user a role of owner (with ID 3) is not allowed
```

Comment:
Will the owner role always have ID "3" - it's not very clear from this that "3" means owner

Reply:
Would it be worth creating an enum for the roles? Then we can refer to them by role name rather than the ID, meaning fewer places to change should the role ID change